pkg/innerring/blocktimer.go

```diff
@@ -1,92 +0,0 @@
-package innerring
-
-import (
-	"github.com/nspcc-dev/neofs-node/pkg/innerring/processors/settlement"
-	"github.com/nspcc-dev/neofs-node/pkg/morph/client/container"
-	"github.com/nspcc-dev/neofs-node/pkg/morph/event"
-	"github.com/nspcc-dev/neofs-node/pkg/morph/timer"
-	"go.uber.org/zap"
-)
-
-type (
-	epochState interface {
-		EpochCounter() uint64
-		EpochDuration() uint64
-	}
-
-	subEpochEventHandler struct {
-		handler     event.Handler // handle to execute
-		durationMul uint32        // X: X/Y of epoch in blocks
-		durationDiv uint32        // Y: X/Y of epoch in blocks
-	}
-
-	newEpochHandler func()
-
-	epochTimerArgs struct {
-		l *zap.Logger
-
-		newEpochHandlers []newEpochHandler
-
-		cnrWrapper *container.Client // to invoke stop container estimation
-		epoch      epochState        // to specify which epoch to stop, and epoch duration
-
-		stopEstimationDMul uint32 // X: X/Y of epoch in blocks
-		stopEstimationDDiv uint32 // Y: X/Y of epoch in blocks
-
-		basicIncome subEpochEventHandler
-	}
-)
-
-func newEpochTimer(args *epochTimerArgs) *timer.BlockTimer {
```

Comment:
Not sure it was worth renaming it (at least not in a separate commit), makes diff larger on GH.

Reply:
> Not sure it was worth renaming it

renaming "block" timer file that does not count blocks is wrong? why? i do like renaming usually, but there is literally no "block timers" anymore in the repo